app/views/partials/_comment.html.erb

```diff
@@ -0,0 +1,9 @@
+<div>
+  <div>
+   <% (1..items).each_with_index do |index| %>
+    <p>
+      Username: Comment <%= index %>
+    </p>
+    <% end %>
+  </div>
```

Comment:
### Requirement
- [ ] Nice to add comments on the show page but comments should come from the model. You should fix it.